build.gradle

```diff
@@ -325,7 +325,7 @@ dependencies {
   implementation group: 'org.springframework', name: 'spring-beans', version: '5.3.23'
   implementation group: 'org.apache.poi', name: 'poi', version: '5.2.2'
   implementation group: 'org.apache.poi', name: 'poi-ooxml', version: '5.2.3'
-  implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6'
+  implementation group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.8-CIV-6069'
```

Comment:
As you stated we are aware that this will have to change.

Code is approved for me. I'll be marking it as "Changes required" just as a gate to prevent any mistake to happen on this front.